Makefile

```diff
@@ -587,6 +587,7 @@ e2e-helm-deploy-ratify:
 	--set notationCerts[0]="$$(cat ~/.config/notation/localkeys/ratify-bats-test.crt)" \
 	--set cosignKeys[0]="$$(cat .staging/cosign/cosign.pub)" \
 	--set cosign.key="$$(cat .staging/cosign/cosign.pub)" \
+	--set cosign.tLogVerify=false \
```

Comment:
Curious why our make file setup doesn't align with the defaults? do we have test coverage for default.

Reply:
`tLogVerify` means that when we sign (key or keyless), the signature is uploaded to a transparency log (rekor public good instance usually). Our tests do offline signing/verification for the most part. The existing cosign keyless test does test this scenario. I added another test for keyless check with the new trust policy way.

Reply:
from a test coverage perspective, i thinks it best cover the default install flags if possible.

Reply:
As discussed in PR meeting, I will investigate what is involved to switch tests to manually disable tLogVerify via template (likely not trivial). And then check to confirm tests will fail if tLogVerify is set to true.

Reply:
@susanshi I ran a test to test if the chart `tLogVerify` is not set correclty, what happens. As expected, the test will fail which is good. 
![image](https://github.com/deislabs/ratify/assets/12980641/4a18bffd-2a0e-4556-8669-3a3a21728ed9)